args.c

```diff
@@ -70,13 +70,19 @@ void parse_args(int argc, char* argv[])
 		if(strcmp(argv[i], "-v") == 0 ||
 		   strcmp(argv[i], "version") == 0)
 		{
-			const char * version = GIT_VERSION;
-			// Check if version is of format vX.YY
-			// '.' can never be part of a commit hash
-			if(strstr(version, ".") != NULL)
+			const char * commit = GIT_HASH;
+			const char * tag = GIT_TAG;
+			if(strlen(tag) > 1)
+			{
 				printf("%s\n",GIT_VERSION);
```

Comment:
It might make more sense to print out `GIT_TAG` as that's the one we're checking, and it will be the same as `GIT_VERSION`.

Reply:
It makes sense to print `GIT_VERSION` here. I just saw that when compiling locally, in contrast to on the CI system, the previous tag is still known:
```
root@raspberrypi:/home/pi/FTL# ./pihole-FTL tag
v2.13.1
root@raspberrypi:/home/pi/FTL# ./pihole-FTL version
v2.13.1-63-gd0d0dfe
```